### PR TITLE
Update Generic SDK Errors with Platform Errors

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/12097/update-generic-errors-with-actual-platform-errors-in-mobile-sdks-v10

## Summary

This ticket changes the default error we show, and replace that with the actual platform code that we get from the api. We allow localization by allowing partners to overide the default empty strings defined on the sdk with the specific messages they would like to show their users

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Util.swift [hunk 1]
```diff
@@ -54,3 +54,65 @@ extension String {
...
+            return ("Si.Error.Message.2201.2301.2401", message)
+        } else if notAuthorized.contains(code) {
+            return ("Si.Error.Message.2205.2405", message)
+        } else if missingParameters.contains(code) {
+            return ("Si.Error.Message.2213.2413", message)
+        }
+        
+        switch code {
+        case "2203":
+            return ("Si.Error.Message.2203", message)
```
Comment (vanshg) [inline Sources/SmileID/Classes/Util.swift:75]: We should be able to construct this dynamically, something like ("Si.Error.Message.\(code)", message)

Comment (vanshg) [reply Sources/SmileID/Classes/Util.swift]: @jumaallan I see that we are still doing a mapping, instead of just constructing the resource string dynamically

Comment (jumaallan) [reply Sources/SmileID/Classes/Util.swift]: Pushed that fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
 
 ## 10.1.5
 
+#### Added
+
+* Update generic errors with actual platform errors
+
```
Comment (vanshg) [inline CHANGELOG.md:14]: This changelog entry needs to be for a new version, not for the already release 10.1.5

**Review by vanshg — APPROVED** [review]
